shell.nix

```diff
@@ -34,10 +34,9 @@ pkgs.stdenv.mkDerivation {
     python
     python3
     time
-    unzip
     wget
-    zip
     z3
+    zstd
```

Comment:
Not sure if we should add *tar* here, but I couldn't find it in nixpkgs, so I suspect it's always available somehow.

Reply:
`tar` is available as `gnutar` fwiw, but yeah, since it's  part of the `stdenv` ([src](https://nixos.org/nixpkgs/manual/#sec-tools-of-stdenv)) it's probably not required here.

Reply:
I think that's the stdenv when building - e.g. it wouldn't make sense to have *make* available everywhere, and AFAICT it isn't.

Reply:
this nix-shell is produced using `stdenv.mkDerivation`, so I would assume that it inherits the packages available in the `stdenv`?

Reply:
That's a good point.